README.md

```diff
@@ -20,37 +18,42 @@ _"An interoperable communication library for the spatial internet."_
 - [Build Instructions](#-build-instructions)
 - [Usage](#%EF%B8%8F-usage)
 - [Tenant IDs](#-tenant-ids)
-- [Api Documentation](#-api-documentation)
+- [API Documentation](#-api-documentation)
 - [Contributions](#%EF%B8%8F-contributions)
 - [License](#%EF%B8%8F-license)
 
-****
+---
 
 ## Background
+
 Interoperability is the foundation of the spatial internet. We need experiences to work across multiple technologies in harmony, bridging applications and devices so that we can move beyond a series of disconnected islands.
 
 <p align="center">
     <img src="https://github.com/magnopus-opensource/connected-spaces-platform/assets/99482500/65fa6a8f-f9a4-45d7-9bd4-731d505c7711">
 </p>
 
-The Connected Spaces Platform provides developers across a range of languages and platforms with a means to build *interoperable, cross-reality, multi-user applications*.
+The Connected Spaces Platform provides developers across a range of languages and platforms with a means to build _interoperable, cross-reality, multi-user applications_.
 
 It comes in the form of an open-source client-side library, effectively encoding (in binary form) the standards by which applications can interoperably communicate with one another. Standards that we can all contribute to.
 
-****
+---
 
 ## 📦 Install
-NPM packages are available for Web and Unity and can be found in [Packages](https://github.com/orgs/magnopus-opensource/packages?repo_name=connected-spaces-platform).
 
-- Web Installation:  `npm install @magnopus-opensource/connected-spaces-platform.web`
+NPM packages are available for Web and Unity and can be found on npmjs.
 
-- Unity Installation: `npm install @magnopus-opensource/connected-spaces-platform.unity.core`
+- Web: https://www.npmjs.com/package/connected-spaces-platform.web
+
+- Unity: https://www.npmjs.com/package/connected-spaces-platform.unity.core
```

Comment:
Can we add the following after the URL:

(for more information on Unity pacakge installation, head over [here](https://github.com/magnopus-opensource/connected-spaces-platform/wiki/Using-CSP-For-Unity))

Reply:
![image](https://github.com/magnopus-opensource/connected-spaces-platform/assets/100562726/abf0df75-0240-48fb-bbae-136eb940f6dd)

Thoughts?

Reply:
Perfect

Reply:
Although end the sentence with a full stop please.

Reply:
Done